assets/src/utils/common_utils.py

```diff
@@ -249,13 +252,44 @@ def convert_txt(self):
         )
         return open(tmp_file_name, "rb").read()
 
+    def convert_html(self) -> bytes:
+        minio_config = create_minio_config()
+        minio_client = minio.Minio(**minio_config)
+        pdf_from_html = self.convert_to_pdf()
+        output_pdf_path = (
+            f"files/{self.new_file.id}/" f"{self.new_file.id}.pdf"
+        )
+        minio_client.put_object(
+            bucket_name=self.bucket_storage,
+            object_name=output_pdf_path,
+            data=pdf_from_html,
+            length=len(pdf_from_html),
+        )
```

Comment:
Are you sure this file should be upload to minio directly?
In this case 'assets' microservice will have no idea about it and it will not be shown on UI

Reply:
It's previously written and described for assets as a blank in is_blank_is_created() function.